timescaledb/how-to-guides/hyperfunctions/function-pipelines.md

```diff
@@ -685,20 +745,120 @@ average on each of two dimensions. The available statistical aggregates are:
 |`x_intercept()`|The x intercept of the least squares fit line|
 
 ### Time-weighted averages aggregates
+
 The `average()` accessor can be called on the output of a `time_weight()`. For
 example:
+
 ```sql
 SELECT time_weight('Linear', ts, val) -> average()  FROM measurements;
 ```
 
 ### Approximate count distinct aggregates
+
 This is an approximation for distinct counts. The `distinct_count()` accessor
 can be called on the output of a `hyperloglog()`. For example:
+
 ```sql
 SELECT hyperloglog(device_id) -> distinct_count() FROM measurements;
 ```
 
+## Formatting timevectors
+
+You can turn a timevector into a formatted text representation. There are two
+functions for turning a timevector to text:
+
+*   [`to_text`](#to-text), which allows you to specify the template
+*   [`to_plotly`](#to-plotly), which outputs a format suitable for use with Plotly
```

Comment:
@thatzopoulos which plotly are we talking about here? python plotly, r plotly, rust plotly, plotly the app, plotly something else entirely?

Reply:
Specifically the plotly json chart schema : https://plotly.com/chart-studio-help/json-chart-schema/